client/my-sites/site-settings/podcasting-details/no-permissions.jsx

```diff
@@ -0,0 +1,22 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+function PodcastingNoPermissionsMessage( { translate } ) {
+	return (
+		<div className="podcasting-details__no-permissions">
+			<p>
+				{ translate(
+					"Oops! You don't have permission to manage Podcasting. " +
+						"If you think you should, contact this site's administrator."
+				) }
+			</p>
```

Comment:
Wording suggestions:

- Two paragraphs, one for each sentence
- First paragraph: "Oops! You don't have permission to manage Podcasting settings on this site."
- Second paragraph: "Try changing to a different site or contacting this site's administrator."

I think going to the wrong site is probably the most common case here - it's how I found this bug.